FalconHose support.

sirsean opened the pull request:
Added a parser for FalconHose JSON events, an alert adapter that
understands that all FalconHose events are also alerts, and a topology
to run them.

sirsean committed:
FalconHose support.

Added a parser for FalconHose JSON events, an alert adapter that
understands that all FalconHose events are also alerts, and a topology
to run them.

reaperhulk commented on OpenSOC-Alerts/src/main/java/com/opensoc/alerts/adapters/FalconHoseAlertAdapter.java:
```diff
@@ -0,0 +1,75 @@
+        String dest = "unknown";
+
+        if (content.containsKey("ip_src_addr")) {
+            source = content.get("ip_src_addr").toString();
```
It looks like source is ignored -- is there an `alert.put` missing?

sirsean replied:
Yeah, I didn't know whether the `designated_host` should be the source or dest IP. (The other alert adapters look up a "whitelist" in HBase which I don't understand what it is or how it gets populated.)

I don't know if we have thoughts about whether it should be the source or dest but for now I'll just kill this variable.

reaperhulk commented on OpenSOC-MessageParsers/src/main/java/com/opensoc/parsing/parsers/BasicFalconHoseParser.java:
```diff
@@ -0,0 +1,88 @@
+
+		String raw_message = null;
+
+		try {
```
Looks like indentation levels are a bit screwy in this method.

sirsean committed:
PR fixes.

Removed unused "source" variable in FalconHoseAlertAdapter. Fixed
wacky indentation in the BasicFalconHoseMessageParser.
